tools/cell_census_builder/scripts/release/census_summary.py

```diff
@@ -28,3 +30,64 @@
     stats_df = pd.DataFrame(stats, columns=["organism", "attribute", "unique count"])
     display_stats_df = pd.pivot(stats_df, index=["organism"], columns=["attribute"], values=["unique count"])
     print(display_stats_df)
+
+    # Calculate the diff with respect to the previous version (if specified)
+
+    if previous_census_version is not None:
+        previous_census = cell_census.open_soma(census_version=previous_census_version)
+
+        prev_datasets = previous_census[CENSUS_INFO_NAME]["datasets"].read().concat().to_pandas()
+        curr_datasets = census[CENSUS_INFO_NAME]["datasets"].read().concat().to_pandas()
+
+        # Datasets removed, added
+        curr_datasets_ids = set(curr_datasets["dataset_id"])
+        prev_dataset_ids = set(prev_datasets["dataset_id"])
+
+        added_datasets = curr_datasets_ids - prev_dataset_ids
+        removed_datasets = prev_dataset_ids - curr_datasets_ids
+        if added_datasets:
+            print("Datasets that were added")
+            for d in added_datasets:
+                print(d)
```

Comment:
out of curiosity, was there a format requested for this report?  I suspect that dataset ID would be complemented by the dataset title, and maybe collection title?  @pablo-gar - opinions on how fancy you want this to be?

Reply:
No particular format requests, but they should be easy enough to accommodate. I can add the dataset title and collection title immediately.